src/schema/v2/me/collection.ts

```diff
@@ -16,6 +24,54 @@ export const Collection: GraphQLFieldConfig<any, ResolverContext> = {
     description: "A collection of artworks",
     fields: () => ({
       ...InternalIDFields,
+      artworksConnection: {
+        type: artworkConnection.connectionType,
+        args: {
+          ...pageable(),
+          sort: {
+            type: CollectionSorts,
+            defaultValue: CollectionSorts.getValue("SAVED_AT_DESC")!.value,
+          },
+        },
+        resolve: async (parent, args, context, _info) => {
+          const { collectionArtworksLoader } = context
+          if (!collectionArtworksLoader) return null
+
+          const { id, user_id } = parent
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          const gravityOptions = {
+            page,
+            size,
+            user_id,
+            private: true,
+            sort: args.sort,
+            total_count: true,
+          }
+
+          try {
+            const { headers, body } = await collectionArtworksLoader(
+              id,
+              gravityOptions
+            )
+
+            const totalCount = parseInt(headers["x-total-count"] || "0", 10)
```

Comment:
I've observed that this count at the artworks level (`me.collection.artworksConnection.totalCount`) can diverge from the one on  the collection level (`me.collection.artworksCount`).

I suspect that's because up in Gravity we don't check whether artworks are published or not when we derive these at the collection level:

https://github.com/artsy/gravity/blob/0df9bf9a8414389d616b043743352a821ed1c75f/app/models/domain/collection.rb#L9-L19

Perhaps a new `published_artworks_count` is called for, in the Gravity collection response?